generative_ai/context_caching/list_content_caches.py

```diff
@@ -15,15 +15,11 @@
 
 from typing import List
```

Comment:
Nit: As per [PEP 585](https://peps.python.org/pep-0585) it's now supported and recommended to use `list` instead of `typing.List` for typing.

For example, `List[str]` would be `list[str]`.

You might need to add `from __future__ import annotations` to work on Python < 3.9

Reply:
@davidcavazos , thanks for the nit! This is indeed a valid suggestion. I knew about this PEP and moving toward using the standard collection, but I didn't realize that the imports from `typing` are already deprecated from **3.7**. This totally makes sense, especially since the lowest we support is 3.8. Updated to use `annotations`!